app.js

```diff
@@ -1,16 +1,21 @@
-const express = require('express');
-const router = require('./router');
+'use strict';
 
-// constraints
-const PORT = 8081;
+const express = require('express');
+const cors = require('cors');
+const router = require('./routes/router');
 
-// app
+/**
+ * App
+ */
 const app = express();
+app.use(cors());
 
 // all of our routes will be prefixed with /api
 app.use('/api', router);
 
-// START THE SERVER
-// =============================================================================
+/**
+ * Start the server
+ */
+const PORT = 8081;
```

Comment:
```suggestion
const PORT = process.env.PORT || 8081;
```
make 8081 default if a port environment variable is not set

Reply:
Where do we define the port environment variable?